packages/webapp/src/App.tsx

```diff
@@ -100,6 +100,11 @@ const App = () => {
         >
             <SWRConfig
                 value={{
+                    refreshInterval: 15 * 60000,
```

Comment:
Can this be overridden on the activity page to a shorter value? It has been quite nice to have refreshes on that page . Temporal does a 15 second refresh FWIW

![image](https://github.com/NangoHQ/nango/assets/1724137/e8286ea9-c97e-427f-bf8c-fc9c2bcb1c3d)

Reply:
it's configurable at the call level yes. Activity page is not using SWR yet no?

Reply:
nevermind it does, changing that 👍🏻